Camera/CameraAlvium/camera_alvium.cc

```diff
@@ -8,6 +8,9 @@
 #include <span>
 namespace camera
 {
+
+#define VMB_ERROR(cond) cond != VmbErrorType::VmbErrorSuccess
```

Comment:
Maybe add parenthesis around cond and the whole macro for consistency and expansion safety

Reply:
I also think this might be a good thing to do

Reply:
I also think this might be a good thing to do

Reply:
But you reviewed in 1 minutem are you sure that you read all the code correclty ???

Reply:
gnngngn ( ok i do it)